.github/workflows/latex-build.yml

```diff
@@ -0,0 +1,49 @@
+name: Build Beamer Presentation
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Místo přímo pushe bych dal on tag

Reply:
```yml
on:
  push:
    branches:
      - main
    paths:
      - 'presentation/prezentace.tex'
```

Dal jsem to na změnu souboru, je to okey?

Reply:
Už jsem to předělal na on tag.